### DISCUSSION TITLE
LCMS-IT-TOF and Mzmine 3

### DISCUSSION THREAD

**Discussion by ThaoPDoan** [mzmine/mzmine#1104] [General]
How can I get MS/MS compared across a batch? Also just to clarify, is it MS2 is the MS/MS data for LCMS. (I am using LCMS-IT-TOF instrument)

**Comment by robinschmid** [comment]
You could run feature finding in MZmine and then apply Feature based molecular networking or Ion Identity Molecular Networking on GNPS. There is a ton of documentation and videos out there. This will compare all your MS2 spectra against each other

**Comment by ThaoPDoan** [comment]
Thank for your suggestions! But, there is one thing to note is that, we dont have ion mobility within the instrument, so i want to make sure if it also works for Feature based molecular networking or Ion Identity Molecular Networking on GNPS.


Best,
Thao.
